x/crosschain/keeper/cctx_orchestrator_validate_outbound.go

```diff
@@ -381,7 +409,10 @@ func (k Keeper) processFailedOutboundV2(ctx sdk.Context, cctx *types.CrossChainT
 		cctx.GetCurrentOutboundParam().TxFinalizationStatus = types.TxFinalizationStatus_Executed
 	case types.CctxStatus_PendingRevert:
 		cctx.GetCurrentOutboundParam().TxFinalizationStatus = types.TxFinalizationStatus_Executed
-		cctx.SetAbort("aborted while processing failed outbound", "outbound and revert failed")
+		cctx.SetAbort(types.StatusMessages{
+			StatusMessage:      "revert failed",
+			ErrorMessageRevert: "outbound and revert failed",
```

Comment:
This is the status message, `ErrorMessageRevert` should give reason why revert failed

Reply:
Same comment as above , resolving this to continue discussion there